course_content/notebooks/dask_intro.ipynb

```diff
@@ -831,6 +831,94 @@
    "cell_type": "raw",
    "metadata": {},
    "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using `filter` as a Constraint for Iris Load with Dask\n",
+    "\n",
+    "We have seen how to load a subset of data by passing a `constraint` to `iris.load` when loading from `files`. We have also seen how to do this by passing lists of filenames and constraints to `delayed(iris.load)`, generating a `dask.bag`. Here we will use the `map` and `filter` functionality of `dask.bag` with `iris.load` to subset data using a phenomenon constraint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we should define what we mean when we talk about `map` and `filter`:\n",
```

Comment:
Given that you've stuck a colon at the end of this line, could you put the two following definitions as list items as well, please.